lang/lualanes/Makefile

```diff
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=lualanes
-PKG_VERSION:=3.13.0
-PKG_RELEASE:=2
+PKG_VERSION:=3.16.2
+PKG_RELEASE:=3
```

Comment:
Why 3?

Reply:
I was under the impression it was proper to increment the PKG_RELEASE for each new version. What would you suggest it be instead of 3?

Reply:
If there is a new upstream release, then it should be resetted to 1.

Reply:
Okay, thank you for making me aware of this. I will update the PR.

Reply:
It is also explained in the contributing.md file, which we do have in this repository or you can also see it in the OpenWrt documentation.